devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy

```diff
@@ -41,8 +41,8 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "FLOALT panel WS 30x30", deviceJoinName: "IKEA FLOALT Panel", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-4000K"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "SURTE door WS 38x64", deviceJoinName: "IKEA SURTE Panel", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-4000K"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "JORMLIEN door WS 40x80", deviceJoinName: "IKEA JORMLIEN Panel", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-4000K"
-		fingerprint profileId: "C05E", deviceId: "0220", manufacturer: "AduroSmart Eria", model: "ZLL-ColorTemperature", deviceJoinName: "Eria Color temperature light"
-		fingerprint profileId: "C05E", manufacturer: "Ubec", model: "BBB65L-HY", deviceJoinName: "XLSmart E27 Light Bulb"
+        //AduroSmart
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF, 0019", outClusters: "0019", deviceId: "0220", manufacturer: "AduroSmart Eria", model: "ZLL-ColorTemperature", deviceJoinName: "Eria ZLL Color Temperature Bulb"
```

Comment:
@adurosmart can you rebase the changes on the latest `master` branch so the existing fingerprints aren't removed? Also what is the color temperature range of this device?

Reply:
The color temperature range of our equipment is 2200-6500. How to describe it in the code?

Reply:
@dkirker do we have UI metadata for that range?

Reply:
can give me a sample？

Reply:
You'd need to add `mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-6500K"` to the fingerprint but that won't work currently because "generic-color-temperature-bulb-2200K-6500K" doesn't currently exist. I'll have to check to see if we can get this added. By default it will support 2700-6500.

Reply:
Go ahead and add it, and I'll create the metadata.

Reply:
The metadata for generic-color-temperature-bulb-2200K-6500K has been created and exists on production.

Reply:
Thanks @dkirker !

@adurosmart that means you can add `mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-6500K"` to the end of your fingerprint

Reply:
Thanks！
I rebuilt fingerprint and commited it.